index.html

```diff
@@ -650,7 +443,12 @@ <h2>Resource proof from a destination's point of view</h2>
                 If it's the case, we only want to process the first of these two events and discard the other one.
             </p>
         </div>
-        <button class="collapsible">ResourceProof_Cancel</button>
+        <button class="collapsible">ResourceProof_Start</button>
+        <div class="content">
+            <p>Provides an external entry point to start with a new available node.
+               Called when we accept a new candidate.
```

Comment:
I don't see this being called from anywhere. Also, I don't see it in this flow which confuses me a bit.

Reply:
You are correct, I forgot about it, It is no longer needed.